src/index.js

```diff
@@ -23,6 +23,7 @@ const MarkupConfig = require('./MarkupConfig');
 const Condition = require('./Condition.js');
 const wrap = require('./wrap.js');
 const processQueue = require('./process-queue.js');
+const bodyData = require('./body-data-wrapper');
```

Comment:
are you sure this should be:
```suggestion
const { bodyData } = require('./body-data-wrapper');
```